### PR TITLE
Minor lint tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -73,8 +74,7 @@ server.register(require('inert'), function (err) {
       };
       helpers.generate(options, function (result) {
         reply(
-          JSON // jshint ignore:line
-            .stringify(result)
+          JSON.stringify(result)    // jshint ignore:line
```
Comment (fmarier) [inline index.js]: Do we still need the `// jshint ignore:line`?

Comment (XhmikosR) [inline index.js]: Unfortunately, yes. Your JSHint options are weird :/

`enforceall` is deprecated, and in theory JSON should be defined...

**Comment by XhmikosR** [comment]
@fmarier: I pushed a patch which adds/cleans up the rules. Let me know what you think and I'll squash it with the first.

**Comment by fmarier** [comment]
Looks good.

**Comment by XhmikosR** [comment]
Ouch :/ I wanted to rebase this, but well.

**Pull request merged**